controller/system_backup_controller.go

```diff
@@ -834,7 +834,7 @@ func (c *SystemBackupController) WaitForVolumeBackupToComplete(backups map[strin
 		for name := range backups {
 			// Retrieve the latest backup
 			var backup *longhorn.Backup
-			backup, err = c.ds.GetBackup(name)
+			backup, err = c.ds.GetBackupRO(name)
```

Comment:
What's the purpose of the change?

Reply:
We only read `backup` and will not modify it so use `GetBackupRO` here.